tox.ini

```diff
@@ -11,7 +11,7 @@ basepython =
 setenv = VIRTUAL_ENV={envdir}
 usedevelop = True
 install_command = pip install -U {opts} {packages}
-deps = -r{toxinidir}/test-requirements.txt
+deps = -r{toxinidir}/test-requirements.txt -r{toxinidir}/docs-requirements.txt
```

Comment:
CI fails because tox considers `-r{toxinidir}/test-requirements.txt -r{toxinidir}/docs-requirements.txt` as a single string and pep fails to find a path like that.
so, it has to be like this:
```
deps = -r{toxinidir}/test-requirements.txt 
       -r{toxinidir}/docs-requirements.txt
```
here's similar multiline definition https://github.com/cloudevents/sdk-python/blob/master/tox.ini#L6